java/ingest/src/test/java/sleeper/ingest/job/IngestJobRunnerTest.java

```diff
@@ -204,30 +192,31 @@ private List<String> writeParquetFilesForIngest(
         return files;
     }
 
-
     private void consumeAndVerify(Schema sleeperSchema,
+                                  List<IngestJob> ingestJobList,
```

Comment:
This is only ever called with a singletonList. We can just pass in an IngestJob instead.